packages/nextjs/test/integration/test/browser.js

```diff
@@ -0,0 +1,151 @@
+const fs = require('fs').promises;
+const { createServer } = require('http');
+const { parse } = require('url');
+const path = require('path');
+
+const next = require('next');
+const puppeteer = require('puppeteer');
+
+const {
+  isSentryRequest,
+  isEventRequest,
+  DEBUG_MODE,
+  logDebug,
+  extractEventFromRequest,
+  extractEnvelopeFromRequest,
+  isSessionRequest,
+  isTransactionRequest,
+} = require('./utils');
+const { log } = console;
+
+/**
+ * Usage: node browser.js <filename>
+ *
+ * ENV Variables:
+ * DEBUG=[bool] - enable requests logging
+ * DEBUG_DEPTH=[int] - set logging depth
+ *
+ * Arguments:
+ * <filename> - filter tests based on filename partial match
+ */
+
+const FILES_FILTER = process.argv[2];
+
+(async () => {
+  let scenarios = await fs.readdir(path.resolve(__dirname, './browser'));
+
+  if (FILES_FILTER) {
+    scenarios = scenarios.filter(file => file.toLowerCase().includes(FILES_FILTER));
+  }
+
+  if (scenarios.length === 0) {
+    console.log('No tests founds.');
+    process.exit(0);
+  } else {
+    if (DEBUG_MODE) {
+      scenarios.forEach(s => console.log(`⊙ Tests Found: ${s}`));
+    }
+  }
+
+  // Silence all the unnecessary server noise. We are capturing errors manualy anyway.
+  if (!DEBUG_MODE) {
+    console.log = () => {};
+    console.error = () => {};
+  }
+
+  const app = next({ dev: false, dir: path.resolve(__dirname, '..') });
+  const handle = app.getRequestHandler();
+  await app.prepare();
+  const server = createServer((req, res) => handle(req, res, parse(req.url, true)));
+
+  const browser = await puppeteer.launch({
+    devtools: false,
+  });
+
+  const success = await new Promise(resolve => {
+    server.listen(0, err => {
+      if (err) throw err;
+
+      const cases = scenarios.map(async testCase => {
+        const page = await browser.newPage();
+        page.setDefaultTimeout(2000);
+
+        if (DEBUG_MODE) page.on('console', msg => logDebug(msg.text()));
+
+        await page.setRequestInterception(true);
+        page.on('request', request => {
+          if (
+            isSentryRequest(request) ||
+            // Used for testing http tracing
+            request.url().includes('example.com')
```

Comment:
`example.com` (`http://example.com` in other places) is required, or can any URL be used? If it has to be this one, is there any other way to use (or modify) it, instead of hardcoding it everywhere?

Reply:
Added missing protocol. Any URL can be used, but it has to match, that's why I used the same one everywhere.